lstchain/data/lstchain_standard_config.json

```diff
@@ -261,7 +262,7 @@
       "tel_id":1,
       "time_sampling_correction_path": null,
       "charge_product":"LocalPeakWindowSum",
-      "charge_median_cut_outliers": [-0.5,0.5],
+      "charge_median_cut_outliers": [-0.9,2],
```

Comment:
why did this change, for the application of the cut also to moon data? we should in this case consider a couple of cases because for standard data we may be missing some outliers because of a stronger cut.

Reply:
Yes.  For that I added the above cut on the number of estimated photo-electrons (npe_median_cut_outliers)

Reply:
and that one is applied exactly how to differentiate between moon and dark data?

Reply:
ok, but based on what? could you please add some comments on how these numbers were obtained?

Reply:
These numbers were obtained on general considerations and experience, they are much less strict that what we had before (thanks to the cut on n_pe),  they ask that a pixel have a signal of at least 10% the camera median because otherwise the cat-B calibration can not be performed with enough precision  (recall that high NSB reduces the gain of a factor ~ 6, for which we an safely perform a Cat-B calibration ) and they must have a signal not  higher than two times the camera median, this is just to detect faulty pixels with uncontrolled high signal (the HV FF is 3% and in case of moon we have now also a FF low HV table). I am afraid one can not put comments in a json file. One should write an internal note with all this inside, let's say that before we start the next mass production I plan to confirm all these cuts with a study on dark and moon data.